diff --git a/lib/cjs/config/index.js b/lib/cjs/config/index.js
index e927b8ce5793968e1c48e4ac1cc7dc44125f3e34..b2ff6e91c7681a55dd15a6b7393cc3f5207b54d4 100644
--- a/lib/cjs/config/index.js
+++ b/lib/cjs/config/index.js
@@ -1,7 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.Config = void 0;
-const client_1 = require("@mysten/sui/client");
+const client_1 = require("@mysten/sui/jsonRpc");
 const pyth_sui_js_1 = require("@pythnetwork/pyth-sui-js");
 const apiEndpoints_1 = require("../constants/apiEndpoints");
 const HERMES_API = "https://hermes.pyth.network";
@@ -10,8 +10,8 @@ const PYTH_STATE_ID = "0x1f9310238ee9298fb703c3419030b35b22bb1cc37113e3bb5007c99
 let apiKey = "";
 let bluefinXApiKey = "";
 let bluefinAggregatorApiKey = "";
-let suiClient = new client_1.SuiClient({
-    url: (0, client_1.getFullnodeUrl)("mainnet"),
+let suiClient = new client_1.SuiJsonRpcClient({
+    url: (0, client_1.getJsonRpcFullnodeUrl)("mainnet"),
 });
 let pythClient = new pyth_sui_js_1.SuiPythClient(suiClient, PYTH_STATE_ID, WORMHOLE_STATE_ID);
 let pythConnection = new pyth_sui_js_1.SuiPriceServiceConnection(HERMES_API);
diff --git a/lib/esm/config/index.js b/lib/esm/config/index.js
index 51f1a50f2b2964a222af6d9eee3d5c7a150ffed4..c4ed2d2a0961f66eb9ef45d06eaa9c9c754cdde7 100644
--- a/lib/esm/config/index.js
+++ b/lib/esm/config/index.js
@@ -1,4 +1,4 @@
-import { SuiClient, getFullnodeUrl } from "@mysten/sui/client";
+import { SuiJsonRpcClient as SuiClient, getJsonRpcFullnodeUrl as getFullnodeUrl } from "@mysten/sui/jsonRpc";
 import { SuiPriceServiceConnection, SuiPythClient, } from "@pythnetwork/pyth-sui-js";
 import { API_ENDPOINTS, DEFAULT_BASE_URL } from "../constants/apiEndpoints.js";
 const HERMES_API = "https://hermes.pyth.network";
